### PR TITLE
PSC Rename

### PR DESCRIPTION
**Pull request by lukehinds**
This is pull request is part of the process to rename the product
security commitee to the security response commmitee.

Please see [the community
issue](https://github.com/kubernetes/community/pull/5597/files)

Please note, may also change minute notes, but on inspection
it helps those minutes make better sense and does not change
the intent of any statements.

Signed-off-by: Luke Hinds <lhinds@redhat.com>

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: communication/slack-config/usergroups.yaml [hunk 1]
```diff
@@ -43,14 +43,14 @@ usergroups:
       - Jason
 
   # Should match the membership of the following teams at all times:
-  # - https://git.k8s.io/security/#product-security-committee-psc
+  # - https://git.k8s.io/security/#security-response-committee-src
```
Comment (cpanato) [inline communication/slack-config/usergroups.yaml:46]: matches with the changes from this PR: https://github.com/kubernetes/security/pull/128

**Review by cpanato — APPROVED** [review]
/lgtm

**Comment by cblecker** [comment]
/hold

@lukehinds Can you please revert the changes to `archive/` and `communication/meeting-notes-archive/`?

**Comment by lukehinds** [comment]
> /hold
> 
> @lukehinds Can you please revert the changes to `archive/` and `communication/meeting-notes-archive/`?

reverted in https://github.com/kubernetes/community/pull/5966/commits/a15718ea92b935d516ea3df0ac2840479a818231

**Comment by lukehinds** [comment]
a big rename merge conflict just landed on this. I am tempted to just make a fresh PR instead.

**Comment by justaugustus** [comment]
> @lukehinds is this still desired? looks like most of the changes were made in #5868 and #6008.

@lukehinds -- Ping to see if this PR is still needed! :)

**Comment by lukehinds** [comment]
> > @lukehinds is this still desired? looks like most of the changes were made in #5868 and #6008.
> 
> @lukehinds -- Ping to see if this PR is still needed! :)

Yep, I need to get this done. Do you mind if I close and make a fresh PR, likely easier then picking through a bucket of merge conflicts?

**Comment by justaugustus** [comment]
> Yep, I need to get this done. Do you mind if I close and make a fresh PR, likely easier then picking through a bucket of merge conflicts?

SGTM. Just tag us when you're ready for review on the next PR!
/close

**Pull request closed without merging**